### ISSUE TITLE
Transifex failing for LRMFE

### ISSUE THREAD

**Issue by hurtstotouchfire** [openedx/frontend-app-learner-record#113]
The LRMFE’s translations job has been failing since October 16th.

See latest failure (Nov 14th): https://tools-edx-jenkins.edx.org/job/translations/job/frontend-app-learner-record-pull_translations/9/:

```
20:51:25 git diff --exit-code ./src/i18n
20:51:25 diff --git a/src/i18n/messages/ar.json b/src/i18n/messages/ar.json
20:51:25 index aa16a3d..463ebfe 100644
20:51:25 --- a/src/i18n/messages/ar.json
20:51:25 +++ b/src/i18n/messages/ar.json
20:51:25 @@ -4,7 +4,7 @@
20:51:25    "partially.completed.status.badge": "Partially Completed",
20:51:25    "link.back.to.records": "Back to My Records",
20:51:25    "records.list.error": "An error occurred attempting to retrieve your program records. Please try again later.",
20:51:25 -  "page.notfound.message": "الصفحة التي تبحث عنها غير متوفرة أو هناك خطأ في نص الرابط. رجاءً تحقق من الرابط و حاول مجدّدًا.",
20:51:25 +  "page.notfound.message": "الصفحة التي تبحث عنها غير متوفرة أو هناك خطأ في العنوان. رجاءً تحقق من العنوان و حاول مجددًا.",
20:51:25    "download.button.default": "Download program record",
20:51:25    "download.button.pending": "Downloading program record",
20:51:25    "download.button.complete": "Download complete",
20:51:25 Makefile:46: recipe for target 'detect_changed_source_translations' failed
20:51:25 make: *** [detect_changed_source_translations] Error 1
20:51:25 2022-11-13 20:51:25,860 INFO 13516 [utils.py:237] - Changes detected for [frontend-app-learner-record]. Pushing them to GitHub and opening a PR.
20:51:25 [transifex-bot-update-translations2022-11-13 30d9e4f] chore(i18n): update translations
20:51:25  1 file changed, 1 insertion(+), 1 deletion(-)
20:51:26 remote: Permission to edx/frontend-app-learner-record.git denied to edx-transifex-bot.
20:51:26 fatal: unable to access 'https://github.com/edx/frontend-app-learner-record.git/': The requested URL returned error: 403
20:51:26 Traceback (most recent call last):
20:51:26   File "transifex/pull.py", line 120, in <module>
20:51:26     pull(
20:51:26   File "transifex/pull.py", line 69, in pull
20:51:26     repo.commit_push_and_open_pr(skip_check_changes)
20:51:26   File "/edx/var/jenkins/jobs/translations/jobs/frontend-app-learner-record-pull_translations/workspace/ecommerce-scripts/transifex/utils.py", line 239, in commit_push_and_open_pr
20:51:26     self.push()
20:51:26   File "/edx/var/jenkins/jobs/translations/jobs/frontend-app-learner-record-pull_translations/workspace/ecommerce-scripts/transifex/utils.py", line 272, in push
20:51:26     subprocess.run(['git', 'push', '-u', 'origin', self.branch_name], check=True)
20:51:26   File "/usr/lib/python3.8/subprocess.py", line 516, in run
20:51:26     raise CalledProcessError(retcode, process.args,
20:51:26 subprocess.CalledProcessError: Command '['git', 'push', '-u', 'origin', 'transifex-bot-update-translations2022-11-13']' returned non-zero exit status 128.
20:51:26 Build step 'Execute shell' marked build as failure
20:51:26 Archiving artifacts
20:51:26 Sending e-mails to: aperture@2u-internal.opsgenie.net
20:51:27 Finished: FAILURE
```

**Comment by hurtstotouchfire** [comment]
From @justinhynes:

> I think this has something to do with the repo transitioning from the edx GitHub org to the openedx GitHub org.

> I assumed I would have to make a change to add the transifex account/bot with permissions to the repo, but I can’t seem to access the settings page of the LR MFE (but I have access to all the other MFE repos in the openedx org).

> I wonder if the edx-admin team hasn’t been added to the MFE’s settings. 

> Otherwise, I don’t think there is anything we can do to fix this ourselves right now. We may have to open up a ticket with tCRIL to (a) restore our admin access to the repo, or (b) have tCRIL fix this issue for us.

**Comment by hurtstotouchfire** [comment]
@nedbat is this something you can fix?

**Comment by nedbat** [comment]
@Carlos-Muniz ?

**Comment by Carlos-Muniz** [comment]
```
20:51:26 remote: Permission to edx/frontend-app-learner-record.git denied to edx-transifex-bot.
20:51:26 fatal: unable to access 'https://github.com/edx/frontend-app-learner-record.git/': The requested URL returned error: 403
```

I think this may be the culprit. I've created https://github.com/openedx/tcril-engineering/issues/567 so tCRIL on-call can fix it.

**Comment by Carlos-Muniz** [comment]
@nedbat @hurtstotouchfire 
edx-transifex-bot should now have the necessary permissions now. Please let me know if this didn't fix it.

**Comment by hurtstotouchfire** [comment]
Here's the new error:
```
16:22:42 Traceback (most recent call last):
16:22:42   File "transifex/pull.py", line 120, in <module>
16:22:42     pull(
16:22:42   File "transifex/pull.py", line 86, in pull
16:22:42     repo.merge_pr()
16:22:42   File "/edx/var/jenkins/jobs/translations/jobs/frontend-app-learner-record-pull_translations/workspace/ecommerce-scripts/transifex/utils.py", line 373, in merge_pr
16:22:42     raise RuntimeError('Pending status checks or other failure prevented this PR from being automatically merged')
16:22:42 RuntimeError: Pending status checks or other failure prevented this PR from being automatically merged
16:22:42 Build step 'Execute shell' marked build as failure
16:22:42 Archiving artifacts
16:22:42 Sending e-mails to: aperture@2u-internal.opsgenie.net
16:22:43 Finished: FAILURE
```

**Comment by Carlos-Muniz** [comment]
@justinhynes Thanks for the ping. 
We've already [given the edx-transifex-bot write access to this repository](https://github.com/openedx/tcril-engineering/issues/567#issuecomment-1333839273). Not sure what else might be the problem, but I suspect more in depth logs will be needed.

**Comment by hurtstotouchfire** [comment]
Is this still failing?

**Comment by justinhynes** [comment]
Our internal Transifex job succeeded on Oct 10th for the first time as of early July. A PR was created and merged successfully: https://github.com/openedx/frontend-app-learner-record/pull/198

However, upon further inspection, it was not merged automatically by the bot. It looks like there was some manual intervention to approve the PR that allowed it to be merged.

IIRC, the desired state is that a human isn't required to pull new translations from Transifex.

There are no pending changes/new translations at this time, so I wouldn't be surprised if it ends up failing again in the future.

**Comment by deborahgu** [comment]
hmm, this should have been fixed by [axim-engineering #940](https://github.com/openedx/axim-engineering/issues/940), but clearly wasn't (the job still requires manual approval as far as I can tell).

**Comment by deborahgu** [comment]
This is gotten fixed in the interim
